gymnasium/wrappers/vector/vectorize_action.py

```diff
@@ -161,10 +161,10 @@ def actions(self, actions: ActType) -> ActType:
         else:
             return deepcopy(
                 concatenate(
-                    self.single_action_space,
+                    self.env.single_action_space,
                     tuple(
                         self.wrapper.func(action)
-                        for action in iterate(self.env.action_space, actions)
+                        for action in iterate(self.action_space, actions)
```

Comment:
I had just looked at the example docstring which shows a batched space rather than a single space so I'm using the batched space version. 
I think the docstring is just incorrect, will update and check the rest.

Reply:
Looks like someone, possibly me, copied the concatenate docstring for some reason, fixing now

Reply:
For the second point, your correct, I'll update